<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LxQtKbIndicatorConfiguration</class>
 <widget class="QDialog" name="LxQtKbIndicatorConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>178</width>
    <height>126</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Keyboard Indicator Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QCheckBox" name="capsLockCB">
     <property name="text">
      <string>Caps Lock</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="numLockCB">
     <property name="text">
      <string>Num Lock</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="scrollLockCB">
     <property name="text">
      <string>Scroll Lock</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>1</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttons</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttons</sender>
   <signal>accepted()</signal>
   <receiver>LxQtKbIndicatorConfiguration</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>97</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttons</sender>
   <signal>rejected()</signal>
   <receiver>LxQtKbIndicatorConfiguration</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>253</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>maximumNetSpeedChanged(QString)</signal>
  <slot>on_typeCOB_currentIndexChanged(int)</slot>
  <slot>on_sourceCOB_currentIndexChanged(int)</slot>
  <slot>on_maximumHS_valueChanged(int)</slot>
  <slot>saveSettings()</slot>
 </slots>
</ui>
